ui/src/app/pages/data/cohort/modifier-page.tsx

```diff
@@ -176,7 +176,7 @@ const dateTooltip = `Dates are consistently shifted within a participant’s rec
 
 const getDefaultFormState = () => {
   const defaultFormState = {
-    name: ModifierType.AGEATEVENT,
+    name: ModifierType.AGE_AT_EVENT as ModifierType,
```

Comment:
When type was unspecified, it was assuming type of "Age_AT_EVENT" rather than ModifierType